system/Cache/Handlers/WincacheHandler.php

```diff
@@ -81,6 +73,8 @@ public function get(string $key)
 	 */
 	public function save(string $key, $value, int $ttl = 60)
 	{
+		$key = static::validateKey($key);
+
 		return wincache_ucache_set($this->prefix . $key, $value, $ttl);
```

Comment:
Remove the duplicate prefix
```suggestion
		return wincache_ucache_set($key, $value, $ttl);
```

Reply:
Whoops! Missed that - incoming.